client/wrappers/timeout/history_generated_test.go

```diff
@@ -107,3 +108,25 @@ func Test_historyClient_CloseShard(t *testing.T) {
 		})
 	}
 }
+
+func Test_historyClient_GetReplicationMessages(t *testing.T) {
+	t.Run("no timeout", func(t *testing.T) {
+		m := history.NewMockClient(gomock.NewController(t))
+		m.EXPECT().GetReplicationMessages(gomock.Any(), gomock.Any()).DoAndReturn(func(ctx context.Context, r *types.GetReplicationMessagesRequest, opt ...yarpc.CallOption) (*types.GetReplicationMessagesResponse, error) {
+			for {
```

Comment:
sorry, maybe just me being dumb, but I am not sure I understand why this is a loop?

Reply:
It's for wait and handle the context timeout.

Reply:
It doesn't matter very much, so this is more a nit comment, but wouldn't jsut the select statement work?

Reply:
I think you are right. We don't have default case so it's not really needed.